src/pages/ReposSearchPage/ReposSearchPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import React from "react";
+import { useEffect } from "react";
+
+import "./ReposSearchPage.css";
+import Button from "@components/Button/Button";
+import Input from "@components/Input/Input";
+import RepoTile from "@components/RepoTile/RepoTile";
+import SearchIcon from "@components/SearchIcon/SearchIcon";
+import GitHubStore from "@store/GitHubStore/GitHubStore";
+import { RepoItem } from "@store/GitHubStore/types";
+
+function ReposSearchPage() {
+  const [repoList, setRepoList] = useState<RepoItem[]>([]);
+  const [value, setValue] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+  const [disabled, setDisabled] = useState(false);
+  const PROFILE = "ktsstudio";
+
+  useEffect(() => {
+    const getRepos = async () => {
+      try {
+        await new GitHubStore()
+          .getOrganizationReposList({
+            organizationName: PROFILE,
+          })
+          .then((repo) => setRepoList(repo.data))
+          .finally(() => {
+            setIsLoading(false);
+            setDisabled(false);
+          });
+      } catch (err) {}
+    };
+    getRepos();
+  }, [value]);
+
+  const mouseChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value);
+  };
+
+  const mouseSearch = () => {
+    const filteredData = repoList.filter((repo) => {
+      return repo.name.toLowerCase().includes(value.toLowerCase());
+    });
+    setRepoList(filteredData);
+    setDisabled(true);
+  };
+
+  return (
+    <div className="wrapper">
+      <div className="search_bar">
+        <Input
+          placeholder="Введите название репозитория"
+          onChange={mouseChange}
+          value={value}
+        />
+        <Button onClick={mouseSearch} disabled={disabled}>
+          <SearchIcon />
+        </Button>
+      </div>
+
+      <div className="repos_list">
+        {repoList.map((repo) => (
+          <React.Fragment key={repo.id}>
+            <RepoTile repo={repo} />
+          </React.Fragment>
```

Comment:
можно просто вот так

```suggestion
            <RepoTile repo={repo} key={repo.id} />  
```